WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2974,6 +2974,11 @@ public void onEditPostPublishedSettingsClick() {
      * EditorFragmentListener methods
      */
 
+    @Override
+    public void updateFeaturedImage(final long mediaId, final boolean imagePicked) {
+        mEditPostSettingsFragment.updateFeaturedImage(mediaId, imagePicked);
```

Comment:
Any particular reason to not reuse the [already defined `setFeaturedImageId`](https://github.com/wordpress-mobile/WordPress-Android/blob/05ddeced286ff7ab0ee67cfe856b178f0af6f0d7/WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java#L2550)? That one calls the same function in `mEditPostSettingsFragment` and also checks if the reference is null, making it slightly more safe.

Reply:
Ah, only my lack of knowledge. :D I hadn't paid much attention to that function within EditPostActivity, I'd been too focused on inspecting the functions under EditPostSettings. I've updated this now and it works great.